src/main/java/com/gtnewhorizons/angelica/glsm/GLStateManager.java

```diff
@@ -121,7 +121,7 @@ public class GLStateManager {
     @Getter protected static final BooleanStateStack rescaleNormalState = new BooleanStateStack(GL12.GL_RESCALE_NORMAL);
 
     @Getter protected static final MatrixModeStack matrixMode = new MatrixModeStack();
-    @Getter protected static final Matrix4fStack modelViewMatrix = new Matrix4fStack(MAX_MODELVIEW_STACK_DEPTH);
+    protected static final Matrix4fStack modelViewMatrix = new Matrix4fStack(MAX_MODELVIEW_STACK_DEPTH);
```

Comment:
Not immediately obvious why it's redundant from the diff

Reply:
`getModelViewMatrix` already exists explicitly

Reply:
https://github.com/GTNewHorizons/Angelica/actions/runs/9968230573/job/27543146104#step:9:615

Reply:
Does it do anything special?  Would probably opt to go the lombok route if not

Reply:
Nope

Reply:
Not seeing this reverted on the GH diff

Reply:
We are keeping the explicit method since its already in use. I removed the lombok \@Getter

Reply:
refactor the recently added method to the lombok one and kill the added one